src/neuromaps_nhp/graph.py

```diff
@@ -0,0 +1,394 @@
+"""Module for managing brain template surface spaces and their transformations."""
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Dict, List, Optional
```

Comment:
Instead of `Dict`, `List`, `Option` use modern syntax (`list[str]`, `dict[str, Any]`, `str | None`)